### PR TITLE
Introduce a third logo color for dev

### PR DESCRIPTION
**Pull request by bhollis**
I have trouble when I have dev and beta both installed - I can't tell them apart. This introduces a new logo color for dev (green).

### CODE REVIEW

File: config/webpack.js [hunk 1]
```diff
@@ -12,7 +12,8 @@ const NotifyPlugin = require('notify-webpack-plugin');
 const ASSET_NAME_PATTERN = 'static/[name]-[hash:6].[ext]';
 
 module.exports = (options = {}) => {
-  const iconFlavor = options.isProd ? 'prod' : 'beta';
+  // TODO: we need a way of specifying the actual prod config!
+  const iconFlavor = options.isProd ? 'beta' : 'dev';
```
Comment (delphiactual) [inline config/webpack.js]: `const iconFlavor = options.isStable ? 'stable' : options.isProd ? 'beta' : 'dev';`
something like this should do it
//cc @bhollis

Comment (bhollis) [inline config/webpack.js]: Right, that assumes that webpack has an `env.stable` that turns into `options.isStable`. I couldn't find any documentation on `env` in the first place. @joshhunt

Comment (delphiactual) [inline config/webpack.js]: http://stackoverflow.com/a/30061249

Comment (joshhunt) [inline config/webpack.js]: `npm run bundle -s -- --env.prod=true` is what sets it at the moment

Comment (joshhunt) [inline config/webpack.js]: `env.blah` lets us set whatever options we want, so we could definitely change this to like `--env.env=dev|beta|prod`

Comment (bhollis) [inline config/webpack.js]: Great, that's what I needed to know!

**Comment by SunburnedGoose** [comment]
Instead of introducing a third color that is bright, can you desaturate the logo for dev?

**Comment by bhollis** [comment]
Sure, I'll try that. The green is awful, but I figure that way people really know it's dev ;-)

**Comment by bhollis** [comment]
Hmm, I can desaturate, but it pretty quickly just looks grey. Chrome will also grey out disabled extensions. Not sure if that's too confusing.

**Comment by bhollis** [comment]
OK, take a look at them now. Less offensive, more... dreary.

**Comment by SunburnedGoose** [comment]
Is this really that important?  Chrome extension is not going to be primary soon...

**Comment by bhollis** [comment]
No, but I'd like to get it over with so I can tell the difference while I'm working. Used to have only Beta and Dev installed, and Dev had the orange logo, so I could tell them apart. Now it's quite confusing.

**Comment by delphiactual** [comment]
@SunburnedGoose, this also effects the branding in the upper left corner... Making telling the difference easy in screenshots...

**Comment by delphiactual** [comment]
I would argue for also changing the DIM color I think the same area...

**Comment by SunburnedGoose** [comment]
I don't want to make a new brand so whatever we go with, I would prefer it not be the DIM logo or something lowkey like the dismal logo.

**Comment by SunburnedGoose** [comment]
Make it an orange square...  Or big D for Developer/DIM :)

**Comment by bhollis** [comment]
This is too much work. I'll just let people guess.

**Pull request closed without merging**